### PR TITLE
Add xslt for transformations before validation

### PR DESCRIPTION
**Pull request by anurag90x**

### CODE REVIEW

File: markup_language/transformer.xslt [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  <xsl:output method="xml" version="1.0"/>
+  <xsl:template match="node() | @*">
+      <xsl:copy>
+        <xsl:apply-templates select="node() | @*"/>
+      </xsl:copy>
+  </xsl:template>
+
+  <xsl:template match="//img/@src">
+    <xsl:attribute name="src">
+      <xsl:if test="substring-after(current(), '/service/uploads') != ''">
```
Comment (markstory) [inline markup_language/transformer.xslt]: Does `current()` point to the first byte of the attribute?

File: tests/test_input_transformer.py [hunk 2]
```diff
@@ -10,3 +13,15 @@ def test_wrap_in_container__adds_divs_if_none_present():
...
+def test_transform_to_valid_xml_string__closing_tags():
+    xml_string = "<div><img src='https://zz.com/service/uploads/a.jpeg'></div>"
+    eq_("<div><img src='https://zz.com/service/uploads/a.jpeg'/></div>",
+        transform_to_valid_xml_string(xml_string))
+
+
+def test_transform_to_valid_xml_string__replace_spaces():
+    xml_string = "<div>&nbsp<p>hello &nbsp</p></div>"
+    eq_("<div>&#160<p>hello &#160</p></div>",
+        transform_to_valid_xml_string(xml_string))
```
Comment (markstory) [inline tests/test_input_transformer.py]: Shouldn't there be a test for Link attribute changes?

File: markup_language/transformer.xslt (hunk 1 above)
Comment (anurag90x) [inline markup_language/transformer.xslt]: It's the value of  the current attribute.

File: tests/test_input_transformer.py (hunk 2 above)
Comment (anurag90x) [inline tests/test_input_transformer.py]: Done

**Comment by markstory** [comment]
Looks good.

**Pull request merged**